internal/controlplane/xdsmgr/xdsmgr.go

```diff
@@ -298,13 +298,14 @@ func (mgr *Manager) nackEvent(ctx context.Context, req *envoy_service_discovery_
 func (mgr *Manager) ackEvent(ctx context.Context, req *envoy_service_discovery_v3.DeltaDiscoveryRequest) {
 	mgr.eventHandler(&events.EnvoyConfigurationEvent{
 		Instance:             mgr.hostname,
-		Kind:                 events.EnvoyConfigurationEvent_EVENT_DISCOVERY_REQUEST,
+		Kind:                 events.EnvoyConfigurationEvent_EVENT_DISCOVERY_REQUEST_ACK,
 		Time:                 timestamppb.Now(),
 		ConfigVersion:        mgr.nonceToConfigVersion(req.ResponseNonce),
 		ResourceSubscribed:   req.ResourceNamesSubscribe,
 		ResourceUnsubscribed: req.ResourceNamesUnsubscribe,
 		TypeUrl:              req.TypeUrl,
-		Message:              fmt.Sprintf("ok %s", req.ResponseNonce),
+		Nonce:                req.ResponseNonce,
```

Comment:
Should we abstract out the messages as status codes similar to `https://golang.org/src/net/http/status.go` ?

Reply:
this field is now only used for errors